### DISCUSSION TITLE
Multi-station set up with more than one VM host

### DISCUSSION THREAD

**Discussion by IngeEclipse** [AzuraCast/AzuraCast#6120] [Q&A]
Hi everyone, we have multiple stations to set up which exceed the recommended number of stations per host (i.e. we have 22).  We would therefore need to set up three VMs to accommodate the number of stations.  Do we set up an instance of Azuracast per VM or does it accommodate a multi-VM environment?

Thanks so much!
Inge

**Comment by druff99** [comment]
have you progressed with this topic?

**Comment by netbladenl** [comment]
For as far as I can tell you should make seperate vms for those stations, with one azuracast instance per server. There's not such a thing as load balancing in azuracast yet.

**Reply by druff99** [reply]
no such thing as native, is there any workaround to administer multiple server of azuracast?
